ledger-service/http-json-oracle/BUILD.bazel

```diff
@@ -30,7 +30,7 @@ da_scala_library(
         "@maven//:org_scalaz_scalaz_core",
     ],
     scalacopts = hj_scalacopts,
-    tags = oracle_tags,
+    tags = oracle_tags + ["canton-ee"],
```

Comment:
@garyverhaegen-da are "canton-ee" excluded on Linux_oracle ?

Reply:
Probably not; I didn't know to look at those.

Reply:
No they don't; this should turn them on:

```patch
diff --git a/ci/build.yml b/ci/build.yml
index 228cf9b559..80da29d2ee 100644
--- a/ci/build.yml
+++ b/ci/build.yml
@@ -249,7 +249,7 @@ jobs:
         bazel test \
           --config=oracle \
           --test_strategy=exclusive \
-          --test_tag_filters=+oracle \
+          --test_tag_filters=+oracle,+canton_ee \
           //...

         oracle_logs=$(Build.StagingDirectory)/oracle-logs
```

Reply:
Possibly need to add `--build_tag_filters=+canton-ee` too, not sure exactly how Bazel resolves that kind of thing.

Reply:
I meant, that your new  `canton-lib-ee` seems to fail on Linux_oracle.

Reply:
👀

Reply:
Right, you need credentials to run it. Let me add a commit with the relevant changes.

Reply:
:bow: